compiler/control/OMROptions.cpp

```diff
@@ -607,6 +607,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"dumpIprofilerMethodNamesAndCounts",  "O\tDebug Printing of Method Names and Persisted Counts.", SET_OPTION_BIT(TR_DumpPersistedIProfilerMethodNamesAndCounts), "F"},
    {"dynamicThreadPriority",              "M\tenable dynamic changing of compilation thread priority", SET_OPTION_BIT(TR_DynamicThreadPriority), "F", NOT_IN_SUBSET},
    {"earlyLPQ",                           "M\tAllow compilations from low priority queue to happen early, during startup", SET_OPTION_BIT(TR_EarlyLPQ), "F", NOT_IN_SUBSET },
+   {"enableAboveTwoGBBar",                "O\tenable allocation of executable memory above the 2gb bar (zOS64)", SET_OPTION_BIT(TR_EnableAboveTwoGBBar), "F"},
```

Comment:
We should change this option to `enableRMODE64`. Please take a look at [1] for reference. Perhaps the description could be "enable resident mode of compiled bodies on z/OS to reside above the the 2-gigabyte bar".

[1] https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.ieab100/resmode.htm